src/rest_api/class-openagenda-controller.php

```diff
@@ -1439,6 +1439,16 @@ public function prepare_item_for_response( $item, $request ) {
 			$item_data[ $meta_key ] = $meta_value;
 		}
 
+		// Create Lat/long based on address data from OSM.
+		if ( ! empty( $item_data['location_address'] ) && ! empty( $item_data['location_zipcode'] ) && ! empty( $item_data['location_city'] ) ) {
+			$address     = $item_data['location_address'] . ', ' . $item_data['location_zipcode'] . ' ' . $item_data['location_city'];
+			$address     = str_replace( ' ', '+', $address );
```

Comment:
What is the reason for this replacement? I suspect URL encoding, but then, should php functions for that not be used instead?